packages/cudajit/cudajit.0.4.0/opam

```diff
@@ -0,0 +1,53 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "Bindings to the `cuda` and `nvrtc` libraries with a unified interface"
+description:
+  "Bindings to manually selected parts of `lcuda` and `lnvrtc`, with a few types and conversion functions to facilitate use."
+maintainer: ["Lukasz Stafiniak"]
+authors: ["Lukasz Stafiniak"]
+license: "MIT"
+tags: ["cuda" "jit" "nvrtc"]
+homepage: "https://github.com/lukstafi/ocaml-cudajit"
+doc: "https://github.com/lukstafi/ocaml-cudajit/blob/master/README.md"
+bug-reports: "https://github.com/lukstafi/ocaml-cudajit/issues"
+depends: [
+  "ocaml"
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
```
(at least) since you use (`let open`, e.g. https://github.com/lukstafi/ocaml-cudajit/blob/main/cudajit.ml#L24C1-L24C21) and `Option.value`